pkgs/development/compilers/jasmin/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchurl
+, unzip
+, jdk
+, ant
+, makeWrapper
+, jre
+}:
+
+let
+  d2u = stdenv.lib.replaceChars ["."] ["_"];
```

Comment:
This isn't used anywhere, is it?
```suggestion
```
(Remove the let/in, too, please.)

Reply:
Oops, that was a leftover from a previous version. Fixed.